<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileShow</class>
 <widget class="QDialog" name="FileShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="FirstProg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="SecondProg">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>381</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="EqualBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>560</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>等价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InequalBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>不等价</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
